react/features/lobby/middleware.ts

```diff
@@ -245,24 +208,45 @@ function _handleLobbyNotification(store: IStore) {
             dispatch(hideNotification(LOBBY_NOTIFICATION_ID));
             dispatch(rejectKnockingParticipant(firstParticipant.id));
         }) ];
-    } else {
+
+        if (showChat) {
+            customActionNameKey.splice(1, 0, 'lobby.chat');
+            customActionHandler.splice(1, 0, () => {
+                dispatch(handleLobbyChatInitialized(firstParticipant.id));
+
+                if (navigator.product === 'ReactNative') {
+                    navigate(chatRoute);
+                }
+            });
+        }
+    }
+
+    if (knockingParticipants.length > 1) {
         descriptionKey = 'notify.participantsWantToJoin';
         notificationTitle = i18n.t('notify.waitingParticipants', {
             waitingParticipants: knockingParticipants.length
         });
         icon = NOTIFICATION_ICON.PARTICIPANTS;
+        concatText = true;
         customActionNameKey = [ 'notify.viewLobby' ];
         customActionHandler = [ () => batch(() => {
             dispatch(hideNotification(LOBBY_NOTIFICATION_ID));
+
+            if (navigator.product === 'ReactNative') {
+                navigate(screen.conference.participants);
```

Comment:
We should probably provide a native implementation for `openParticipantsPane` which does the apprpriate navigation, instead on needed to have platform dependent code here.

Reply:
Ok, I will create an action that does the navigation on native and not depend on platform type.

Reply:
And I think I will have to do the same for the action dispatched when lobby chat button is pressed.

Reply:
👍